draft-ietf-regext-rdap-geofeed.md

```diff
@@ -32,38 +32,101 @@ email = "tomh@apnic.net"
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and
+"OPTIONAL" in this document are to be interpreted as described in BCP
+14 [@!RFC2119] [@!RFC8174] when, and only when, they appear in all
+capitals, as shown here.
 
-Indentation and whitespace in examples are provided only to illustrate element relationships, and are not a REQUIRED
-feature of this protocol.
+Indentation and whitespace in examples are provided only to illustrate
+element relationships, and are not a REQUIRED feature of this
+protocol.
 
-"..." in examples is used as shorthand for elements defined outside of this document.
+"..." in examples is used as shorthand for elements defined outside of
+this document.
 
 # Specification
 
-## Extension
+## Media Type for a Geofeed Link {#media_type_for_a_geofeed_link}
+
+[@?I-D.ietf-opsawg-9092-update] requires a geofeed file to be a UTF-8
+[@!RFC3629] comma-separated values (CSV) file, with a series of "#"
+comments at the end for the optional Resource Public Key
+Infrastructure (RPKI, [@!RFC6480]) signature. At first glance, the
+"text/csv" media type ([@?I-D.shafranovich-rfc4180-bis, section 4])
+seems like a good candidate for a geofeed file, since it supports the
+"#" comments needed for including the RPKI signature.
+
+However, although the CSV geofeed data could be viewed directly by a
+user such that the "text/csv" media type was appropriate, the most
+common use case will involve it being processed by some sort of
+application first, in order to facilitate subsequent address lookup
+operations.  Therefore, using a new "application" media type with a
+"geofeed" subtype ([@!RFC6838, section 4.2.5]) for the geofeed data is
+preferable to using "text/csv".
+
+To that end, this document registers a new "application/geofeed+csv"
+media type in the IANA Media Types Registry (see
+(#media_types_registry)), and a new "+csv" suffix in the IANA
+Structured Syntax Suffixes Registry (see
+(#structured_syntax_suffixes_registry)).
 
-A new RDAP extension "geofeed1" is defined for accessing the geofeed data through RDAP. It updates the IP Network object
-class definition ([@!RFC9083, section 5.4]) to include a new link object for the geofeed file URL in its "links" array
-([@!RFC9083, section 4.2]).
+## Geofeed Link
 
-An RDAP server conforming to this specification MUST include the "geofeed1" extension string in the "rdapConformance"
-array for any lookup or search response containing an IP Network object, as well as in the help response. Here is an
-elided example for this inclusion:
+An RDAP server that hosts geofeed URLs for its IP network objects
+([@!RFC9083, section 5.4]) may include link objects for those geofeed
+URLs in IP network objects in its responses.  These link objects are
+added to the "links" member of each object ([@!RFC9083, section 4.2]).
+An example scenario where a server would not include such a link
+object is where the URL is redacted due to privacy considerations or
+regulatory requirements.
```

Comment:
Do we need to bring back the "redaction" section we had earlier? :) In case, folks end up redacting.

Reply:
Thanks.